pt-PT/Universally Preferable Behaviour/p02_ch07_02_more_challenging_tests_of_upb.md

```diff
@@ -1,59 +1,59 @@
+Nós não temos tempo suficiente aqui para entrar numa discussão completa sobre os direitos dos animais, mas podemos ao menos lidar com a proposição moral: "é mau matar um peixe."
 
-If it is evil to kill fish, then UPB says that anyone or anything that kills the fish is evil. This would include not just fishermen, but sharks as well – since if killing fish is evil, we have expanded our definition of ethical “actors” to include non-human life.
+Se é mau matar um peixe, então o UPB diz que qualquer um ou qualquer coisa que mate um animal é má. Isso iria incluir não só os pescadores, mas também os tubarões - uma vez que matar um peixe é mau, temos que expandir a nossa definição de "atores" éticos para incluir vida não-humana.
 
-It is clear that sharks do not have the capacity to refrain from killing fish, since they are basically eating machines with fins.
+É claro que os tubarões não têm a capacidade de abster-se de matar peixes, uma vez que eles são basicamente máquinas de comer com barbatanas.
 
-Thus we end up with the logical problem of “inevitable evil.” If it is evil to kill fish, but sharks cannot avoid killing fish, then sharks are “inevitably evil.” However, as we have discussed above, where there is no choice – where *avoidability* is impossible – there can be no morality. Thus the proposition “it is evil to kill fish” attempts to define a universal morality that includes non-moral situations, which cannot stand logically.
+Assim nós terminamos com o problema lógico de "mau inevitável." Se é mau matar peixes, mas os tubarões não podem evitar de comer peixes, então os tubarões são um "mau inevitável." No entanto, como discutimos acima, quando não existe escolha - onde *evitabilidade* é impossível - não pode existir moralidade. Assim a proposição "é mau matar um peixe" tenta definir uma moralidade universal que inclui situações não-morais, que não se podem suportar logicamente
 
-Also, the word “fish” remains problematic in the formulation, since it is too specific to be universal. The proper UPB reformulation is: “it is evil for people to kill living organisms.”
+Também, a palavra "peixe" continua a ser a problemática na formulação, já que é muito específica para ser universal. A reformulação apropriada do UPB é: "é mau para pessoas matarem outros organismos seres vivos."  
 
-If, however, it is evil to kill, we again face the problem of “inevitable evil.” No human being can exist without killing other organisms such as viruses, plants, or perhaps animals. Thus “human life” is defined as “evil.” But if human life is defined as evil, then it cannot *be* evil, since avoidance becomes impossible.
+Se, no entanto, é mau matar,  temos que encarar novamente o problema de "mau inevitável." Nenhum ser humano pode existir sem matar outros organismos como vírus, plantas ou talvez animais. Assim, "a vida humana" é definida como "mau." Mas se vida humana é definida como mau, então ela na pode *ser* má, já que evitabilidade se torna impossível.
```

Comment:
There is a small typo, "então ela na pode" should be "então ela não pode"

Reply:
Thank you for the correction.

Reply:
I think there is a way for you to do the corrections, @nunojesus

Reply:
How can I do that?

Reply:
I thought that the utopian moderators would review the translations and change them if necessary.

Reply:
You can commit on the same branch on your fork and the PR will update itself